src/main/kotlin/no/nav/syfo/cronjob/VerifyBehandlereForKontorCronjob.kt

```diff
@@ -76,14 +76,14 @@ class VerifyBehandlereForKontorCronjob(
                         )
 
                         val added = addNewBehandlere(
-                            aktiveBehandlereForKontor.toMutableList().also { it - revalidated },
+                            aktiveBehandlereForKontor.toMutableList().also { it.removeAll(revalidated) },
```

Comment:
Fungerte ikke så godt med den minus syntaksen. Det forårsaket at det ble opprettet duplikater av de som ble revalidert. Dette rydder cronjob'en selv opp i neste gang.